packages/mobile/src/utils/LogUploader.ts

```diff
@@ -49,7 +49,7 @@ export default class FirebaseLogUploader {
     await RNFS.moveFile(logFilePath, tmpFilePath)
```

Comment:
Do we ever remove these tmp files or do they just pile up?

Reply:
I think the temp file gets overwritten every time there is an upload